frontend/src/component/demo/demo-topics.tsx

```diff
@@ -481,18 +484,19 @@ export const TOPICS: ITutorialTopic[] = [
             },
             {
                 href: `/projects/${PROJECT}/features/demoApp.step4`,
-                target: `div[data-testid="FEATURE_ENVIRONMENT_ACCORDION_${ENVIRONMENT}"] .MuiAccordionSummary-expandIconWrapper`,
+                target: `div[data-testid="FEATURE_ENVIRONMENT_ACCORDION_${ENVIRONMENT}"] > div[aria-expanded="false"]`,
```

Comment:
I would try to do what @kwasniew suggested of adding a class to the div, maybe `demoApp.step4` so we can target it more precisely. Maybe a css class is not "the best" but something that specifically indicates this is the demoApp step4 target would be nice